api/upgrader/unitupgrader_test.go

```diff
@@ -55,6 +56,15 @@ func (s *unitUpgraderSuite) SetUpTest(c *gc.C) {
 func (s *unitUpgraderSuite) addMachineApplicationCharmAndUnit(c *gc.C, appName string) (*state.Machine, *state.Application, *state.Charm, *state.Unit) {
 	machine, err := s.State.AddMachine("quantal", state.JobHostUnits)
 	c.Assert(err, jc.ErrorIsNil)
+
+	arch := "amd64"
```

Comment:
Should we change these to use a default arch constant?

Reply:
Sure